plugins/gitfast/update

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 url="https://git.kernel.org/pub/scm/git/git.git/plain/contrib/completion"
-version="2.16.0"
+version="2.28.0"
```

Comment:
This is the only line in this PR that I actually changed, all other files are automatically generated